### PR TITLE
166 evaluation

### LINKED ISSUE #166: Evaluation
Evaluation of the developed product from a user/customer point of view.

### PR DESCRIPTION
**Pull request by kkmp-dk**

### CODE REVIEW

File: project_report/tex/evaluation.tex [hunk 1]
```diff
@@ -1 +1,26 @@
...
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+
+
+\subsection{Development process}
+In the beginning of the development process, the group has held as meeting
+about how the projects should be structured. Building on the experience from
+last semester where Unified Process and Scrum was used, the group has
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Consider removing 'has'.

File: project_report/tex/evaluation.tex [hunk 2]
```diff
@@ -1 +1,26 @@
...
+\subsection{Evaluation from the customer side}
+The group has managed to meet the requirements, set in the initial requirements
+analysis, in the beginning of the project. The created system is able to control
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+
+
+\subsection{Development process}
```
Comment (Kechr) [inline project_report/tex/evaluation.tex:30]: You are stating some facts, but you never evaluate them. 
Consider writing about:
- How did UP and Scrum work for us?
  - Along with all the artefacts from UP (did they help us develop the system)
- How well did we answer the requirements?

File: project_report/tex/evaluation.tex [hunk 3]
```diff
@@ -1 +1,26 @@
 \section{Evaluation}
+The evaluation section describes how the reflections of the goals, set for the
+project, meets the customer demands, as well as how the development process has
+been.
+
+
+\subsection{Evaluation from the customer side}
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: This is validating our solution, not evaluating it. 

Consider writing about:
- Is it user friendly?
  - does it take a long time to understand how to use it etc.
  - Is the design intuiative?
  - How well is the data displayed?

All the good stuff.

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_report/tex/evaluation.tex (hunk 3 above)
Comment (kkmp-dk) [reply project_report/tex/evaluation.tex]: Maybe we have done it now?

File: project_report/tex/evaluation.tex (hunk 2 above)
Comment (kkmp-dk) [reply project_report/tex/evaluation.tex:30]: Maybe we have done it now?

File: project_report/tex/evaluation.tex (hunk 1 above)
Comment (kkmp-dk) [reply project_report/tex/evaluation.tex]: Maybe we have done it now?

File: project_report/tex/evaluation.tex [hunk 4]
```diff
@@ -1 +1,53 @@
...
+The group has managed to meet the requirements, set in the initial requirements
+analysis, in the beginning of the project. The created system is able to control
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: in the beginning -> at the beginning

Consider: is able to -> can

File: project_report/tex/evaluation.tex [hunk 5]
```diff
@@ -1 +1,53 @@
...
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+The design of the dashboard, makes it easy to see what is going on the front-page.
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Remove unnecessary comma.

File: project_report/tex/evaluation.tex [hunk 6]
```diff
@@ -1 +1,53 @@
...
+Every item on the front-page is there nothing is hiding down under a sub menu,
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: sub menu -> sub-menu

File: project_report/tex/evaluation.tex [hunk 7]
```diff
@@ -1 +1,53 @@
...
+beside starting the beer machine, you would need to press the start button,
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: beside -> besides

File: project_report/tex/evaluation.tex [hunk 8]
```diff
@@ -1 +1,53 @@
...
+that will then take you to a start-page where you can chose a recipe, speed and
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: chose -> choose

File: project_report/tex/evaluation.tex [hunk 9]
```diff
@@ -1 +1,53 @@
...
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+The design of the dashboard, makes it easy to see what is going on the front-page.
+Every item on the front-page is there nothing is hiding down under a sub menu,
+beside starting the beer machine, you would need to press the start button,
+that will then take you to a start-page where you can chose a recipe, speed and 
+the amount to produce. The start-page is not that convenient, because the page 
+does not put in the recommend values for the different recipes, but it does
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: recommend -> recommended

File: project_report/tex/evaluation.tex [hunk 10]
```diff
@@ -1 +1,53 @@
...
+show the recommend values for the recipe.
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: recommend -> recommended

File: project_report/tex/evaluation.tex [hunk 11]
```diff
@@ -1 +1,53 @@
...
+For seeing other older batches you will need to navigate to a different page
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: older batches, you will need. | Add a comma after batches

File: project_report/tex/evaluation.tex [hunk 12]
```diff
@@ -1 +1,53 @@
...
+called history.html. You can navigate to this page easy from the front-page by
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: easy -> easily

File: project_report/tex/evaluation.tex [hunk 13]
```diff
@@ -1 +1,53 @@
...
+pressing on the button left side menu bar that takes you to the history page.
+Where you will be able to chose a batch to inspect.
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Remove the full stop, add comma instead.

chose -> choose

File: project_report/tex/evaluation.tex [hunk 14]
```diff
@@ -1 +1,53 @@
...
+The button layout is the same as the front-page, so there should be no learning
+curve there to check out the old batches.
+Overall the group thinks it is a good system for people not familiar with the
+project, because it does comply with the standard UI formalities and there is
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Remove unnecessary comma after project.

Add a comma before and

File: project_report/tex/evaluation.tex [hunk 15]
```diff
@@ -1 +1,53 @@
 \section{Evaluation}
```
Comment (Kechr) [inline project_report/tex/evaluation.tex:1]: There are a lot of personal pronouns in this section. These should be avoided.

File: project_report/tex/evaluation.tex [hunk 16]
```diff
@@ -1 +1,53 @@
...
+The evaluation section describes how the reflections of the goals, set for the
+project, meets the customer demands, as well as how the development process has
+been.
+
+
+\subsection{Evaluation from the customer side}
+The group has managed to meet the requirements, set in the initial requirements
+analysis, in the beginning of the project. The created system is able to control
+the production line, using the dashboard, and is able to collect, display and
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Consider: is able to -> can

File: project_report/tex/evaluation.tex [hunk 17]
```diff
@@ -1 +1,53 @@
...
+the amount to produce. The start-page is not that convenient, because the page 
+does not put in the recommend values for the different recipes, but it does
+show the recommend values for the recipe.
+For seeing other older batches you will need to navigate to a different page
+called history.html. You can navigate to this page easy from the front-page by
+pressing on the button left side menu bar that takes you to the history page.
+Where you will be able to chose a batch to inspect.
+The button layout is the same as the front-page, so there should be no learning
+curve there to check out the old batches.
+Overall the group thinks it is a good system for people not familiar with the
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Consider choosing a different word for good.

File: project_report/tex/evaluation.tex [hunk 18]
```diff
@@ -1 +1,53 @@
...
+project, because it does comply with the standard UI formalities and there is
+almost no learning curve to the dashboard.
+
+
+\subsection{Development process}
+In the beginning of the development process, the group has held as meeting
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: In -> At
as -> a

File: project_report/tex/evaluation.tex [hunk 19]
```diff
@@ -1 +1,53 @@
...
+about how the projects should be structured. Building on the experience from
+last semester where Unified Process and Scrum was used, the group
+decided to use both UP and Scrum again. Every Friday, a Scrum meeting is held,
+to show and discuss what each group member has accomplished. Each task is
+assigned into a two week sprint, which is planned at the appropriate Scrum
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: into -> to
two week -> two-week

File: project_report/tex/evaluation.tex [hunk 20]
```diff
@@ -1 +1,53 @@
...
+meeting.
+During 'matching of expectations', the group agreed upon focusing on the Minimum
+Viable Product, meaning not implementing parts outside of the semester project's
+scope. This has been a more realistic goal in comparison to last semester.
+The use of Scrum has giving a great overview of the project, and has made it
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: giving -> given
Remove unnecessary comma after project

File: project_report/tex/evaluation.tex [hunk 21]
```diff
@@ -1 +1,53 @@
...
+decided to use both UP and Scrum again. Every Friday, a Scrum meeting is held,
+to show and discuss what each group member has accomplished. Each task is
+assigned into a two week sprint, which is planned at the appropriate Scrum
+meeting.
+During 'matching of expectations', the group agreed upon focusing on the Minimum
+Viable Product, meaning not implementing parts outside of the semester project's
+scope. This has been a more realistic goal in comparison to last semester.
+The use of Scrum has giving a great overview of the project, and has made it 
+easier to plan ahead. However, looking back, the group should have scheduled 
+more in beginning. Near the end, the group has had longer work hours.
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: more in beginning -> more in the beginning.
longer -> long

File: project_report/tex/evaluation.tex [hunk 22]
```diff
@@ -1 +1,53 @@
...
+
+The project has been divided into the four phases Unified Process; Inception,
+Elaboration, Construction, and Transition. Each phase has been a fixed boundary
+to plan around, making it easier to schedule, sprints, tasks, and issues.
+Unlike the previous semester, no inception document was created. Instead, the
+initial requirements analysis was incorporated directly into the final report.
+The initial phases of Unified Process could have been more efficient, enabling
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Consider rewriting, as the initial phase of UP cannot be more efficient, but how we worked with it could.

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_report/tex/evaluation.tex [hunk 23]
```diff
@@ -1 +1,51 @@
...
+
+\subsection{Evaluation from the customer side}
+The group has managed to meet the requirements, set in the initial requirements
+analysis, at the beginning of the project. The created system is able to control
+the production line, using the dashboard, as well as collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+The design of the dashboard makes it easy to see what is going on the front-page.
+Every item on the front-page is there, nothing is hiding in a sub-menu.
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: Consider the following;
 "... is there; nothing is..."
 "... is there, and nothing is..."
 "... is there. Nothing is..."

File: project_report/tex/evaluation.tex [hunk 24]
```diff
@@ -1 +1,51 @@
...
+The use of Scrum has given a great overview of the project and has made it 
+easier to plan ahead. However, looking back, the group should have scheduled 
+more in the beginning. Near the end, the group has had long work hours.
+
+The project has been divided into the four phases Unified Process; Inception,
+Elaboration, Construction, and Transition. Each phase has been a fixed boundary
+to plan around, making it easier to schedule, sprints, tasks, and issues.
+Unlike the previous semester, no inception document was created. Instead, the
+initial requirements analysis was incorporated directly into the final report.
+The group could have worked more effeciently in the initial phases of Unified
```
Comment (Kechr) [inline project_report/tex/evaluation.tex]: effeciently -> efficiently

**Review by Kechr — CHANGES_REQUESTED** [review]

**Review by Kechr — APPROVED** [review]

**Pull request merged**